x/metadata/keeper/msg_server.go

```diff
@@ -106,11 +104,11 @@ func (k msgServer) DeleteScopeDataAccess(
 
 	existing, found := k.GetScope(ctx, msg.ScopeId)
 	if !found {
-		return nil, fmt.Errorf("scope not found with id %s", msg.ScopeId)
+		return nil, sdkerrors.ErrInvalidRequest.Wrapf("scope not found with id %s", msg.ScopeId)
```

Comment:
Should keep not found constant with how you are doing it above with `sdkerrors.ErrNotFound.Wrapf`